code/model/UserDefinedForm.php

```diff
@@ -178,10 +178,10 @@ public function getCMSFields()
 
             // get a list of all field names and values used for print and export CSV views of the GridField below.
             $columnSQL = <<<SQL
-SELECT "SubmittedFormField"."Name" as "Name", "SubmittedFormField"."Title" as "Title", COALESCE("EditableFormField"."Sort", 999) AS "Sort"
+SELECT "SubmittedFormField"."Name" as "Name", "EditableFormField"."Title" as "Title", COALESCE("EditableFormField"."Sort", 999) AS "Sort"
```

Comment:
This will now break form results that have changed since they were submitted.

Can you please use COALESCE() to fall back to SubmittedFormField.Title if the original is null?